exercises/concept/story-name/.meta/design.md

```diff
@@ -0,0 +1,28 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach students how to consume and make generic code.
+
+## Learning objectives
+
+- Understand what a type variable is.
```

Comment:
It might be worth calling out `Cmd Msg` and `Cmd msg` somewhere, which is something that confuses a lot of people new to elm I think.

Reply:
Right! that's an important remark!